packages/invoices/src/OutboundShipment/DetailView/DetailViewToolbar.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import {
+  AppBarContentPortal,
+  Box,
+  InputWithLabelRow,
+  BasicTextInput,
+  TabList,
+  Tab,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { CustomerSearchInput } from '../CustomerSearchInput';
+import { OutboundShipment } from './types';
+
+interface OutboundShipmentToolbarProps {
+  draft: OutboundShipment;
+  currentTab: string;
+  onChangeTab: (newTab: string) => void;
+}
+
+export const OutboundShipmentToolbar: FC<OutboundShipmentToolbarProps> = ({
+  draft,
+  onChangeTab,
+  currentTab,
+}) => {
+  const t = useTranslation();
+
+  return (
+    <AppBarContentPortal
+      sx={{
+        display: 'flex',
+        flex: 1,
+        justifyContent: 'center',
+        paddingLeft: '25px',
+      }}
+    >
+      <Box
```

Comment:
In the zeplin design, the inputs are above the tab controls, which allows them to spread horizontally and on smaller screens (like my laptop screen) the 'customer name' label shows on one line.

With the three box layout you have, the inputs and their labels are squished up, and the label wraps to two lines. Could you instead run a full width box for the inputs, then three boxes under that for the 'columns' and 'tab' controls? 

my screen:
<img width="706" alt="Screen Shot 2021-10-18 at 9 28 36 AM" src="https://user-images.githubusercontent.com/9192912/137643779-7bbaebea-e389-434b-a355-9e7bdd5e55d7.png">

zeplin:
<img width="1016" alt="Screen Shot 2021-10-18 at 9 28 25 AM" src="https://user-images.githubusercontent.com/9192912/137643791-d616c421-432e-4617-a870-4738c3ae08c8.png">

Reply:
Oh right ! 

![image](https://user-images.githubusercontent.com/35858975/137646190-020bf892-3516-4e30-bcfd-945373b6eeec.png)

Hope this is a bit better. Need to add the filters and columns buttons?

Reply:
another time.. I suspect that could be a big job